app/scripts/modules/core/src/application/service/applicationDataSource.ts

```diff
@@ -285,6 +286,7 @@ export class ApplicationDataSource implements IDataSourceConfig {
     if (!config.label && this.$filter) {
       this.label = this.$filter('robotToHuman')(config.key);
     }
+    this.label = FirewallLabels.get(this.label);
```

Comment:
am i inferring correctly that this overrides the label defined in `securityGroup.dataSource.ts`?

Reply:
You are correctly inferring.